readme.txt

```diff
@@ -1,64 +1,65 @@
+Tags: Jetpack, WordPress.com, backup, security, related content, CDN, speed, anti spam, sharing, SEO, video
 Stable tag: 4.5
 Requires at least: 4.6
 Tested up to: 4.7
 
-Increase your traffic, view your stats, speed up your site, and protect yourself from hackers with Jetpack.
+Whatever sort of site you’re running, Jetpack keeps your site secure and helps you increase traffic.
 
 == Description ==
 
-Jetpack simplifies managing WordPress sites by giving you visitor stats, security services, speeding up images, and helping you get more traffic. Jetpack is a free plugin.
+Whatever sort of site you’re running, Jetpack’s features are designed to keep your site secure, elevate your brand, and engage with more readers better.
 
-= Traffic Growth & Insights =
-Jetpack tells you how many visits your site gets and helps you **get more traffic** with tools like Related Posts, Publicize, Enhanced Distribution, and Sharing.
+= Traffic and SEO Tools =
+Traffic is the lifeblood of any website. Jetpack includes simple-to-use features that help you automatically promote, share, and optimize your content for maximum readership.
 
-= Security =
-Jetpack **protects your site** against brute force attacks and unauthorised logins. We also monitor your site for downtime and keep your plugins updated.
+= Security and Backup Services =
+You shouldn’t have to worry about data loss, downtime, and hacking. Jetpack monitors your site, protects it from brute force attacks, secures your logins, scans for malicious code, and backs up all your data.
 
-= Image Performance =
-Jetpack automatically **optimizes and speeds up** images using the global WordPress.com content delivery network. This saves you hosting costs by reducing bandwidth.
+= Writing and Content Creation =
+Jetpack offers a free, high-speed CDN for your images and a variety of tools to customize your site to your heart’s content.
 
-= Centralized Management =
-With Jetpack comes a **centralized dashboard** on WordPress.com. Manage plugins and menus, publish posts, and view enhanced site stats for all of your sites.
+= Discussion and Community =
+Create a connection with your readers and keep them coming back to your site with subscriptions, comments, contact forms, and spam filtering.
 
-= A few more things that our users love =
-Jetpack includes other features that help you customize your site including Custom CSS, Contact Forms, Galleries and Carousels, Notifications and Subscriptions, Configurable Widgets, and [many more](https://jetpack.com/features).
+= Paid Services =
+Most of Jetpack's features and services are free. Jetpack also provides advanced security and backup services, video hosting, site monetization, and advanced SEO tools in three [simple and affordable plans](https://jetpack.com/pricing).
 
-= Dedicated Support =
-We have an entire team of happiness engineers ready to help you. Ask your questions in the support forum, or [contact us directly](https://jetpack.com/contact-support).

```

Comment:
While you may not want to promote support anymore, I think it's important to leave a link to the support contact form in the plugin's main page; it's the default tab, it's what you see when you land on the plugin's page or when you open it in the dashboard's Plugins page, and that's where folks will look for links first. If we don't have a link to the contact form on that first page, we should at least have a link to `/support` imo.